README.md

```diff
@@ -1,1633 +1,287 @@
-notation. 
-
-Default value: `"<C-Space>"`
-
-
-## `g:ctrlspace_use_ruby_bindings`
-
-If set to `1`, the plugin will try to use your compiled in Ruby bindings to
-increase the speed of the plugin (e.g. while fuzzy search, since regex
-operations are much faster in Ruby than in VimScript).
-
-> To see if you have Ruby bindings enabled you can use the command `:version`
-> and see if there is a `+ruby` entry. Or just try the following one: `:ruby
-> puts RUBY_VERSION` - you should get the Ruby version or just an error.
-
-Default value: `1`
-
-
-## `g:ctrlspace_glob_command`
-
-If not empty, the provided command will be used to list all files instead of Vim
-`globpath()` function. For example, if you have Ag installed that could be:
-
-```VimL
-if executable("ag") 
-  let g:ctrlspace_glob_command = 'ag -l --nocolor -g ""'
-endif
-```
-
-Default value: `""`
-
-## `g:ctrlspace_use_tabline`
-
-Should **Vim-CtrlSpace** change your default tabline to its own? 
-
-Default value: `1`
-
-
-## `g:ctrlspace_use_mouse_and_arrows_in_term`
-
-Should the plugin use mouse, arrows and `Home`, `End`, `PageUp`, `PageDown`
-keys in a terminal Vim. Disables the `Esc` key if turned on. 
-
-Default value: `0`
-
-
-## `g:ctrlspace_save_workspace_on_exit`
-
-Saves the active workspace (if present) on Vim quit. If this option is set, the
-Vim quit (`Q`) action from the plugin modes does not check for workspace
-changes. 
-
-Default value: `0`
-
-
-## `g:ctrlspace_save_workspace_on_switch`
-
-Saves the active workspace (if present) upon switching to another workspace or
-clearing (closing) the current one. If this option is set, the plugin won't warn
-you about an unsaved workspace.
-
-Default value: `0`
-
-
-## `g:ctrlspace_load_last_workspace_on_start`
-
-Loads the last active workspace (if found) on Vim startup. 
-
-Default value: `0`
-
-
-## `g:ctrlspace_cache_dir`
-
-A directory for the **Vim-CtrlSpace** cache file (`.cs_cache`). By default your
-`$HOME` directory will be used. 
-
-
-## `g:ctrlspace_project_root_markers`
-
-An array of directory names which presence indicates the project root. If no
-marker is found, you will be asked to confirm the project root basing on the
-current working directory. Make this array empty to disable this functionality.
-
-These markes will be also used as a storage for `cs_workspaces` (workspaces of
-the current project) and `cs_files` (cached files of the current project).
-
-Default value: `[".git", ".hg", ".svn", ".bzr", "_darcs", "CVS"]`
-
-
-## `g:ctrlspace_unicode_font`
-
-Set to `1` if you want to use Unicode symbols, or `0` otherwise. 
-
-Default value: `1`
-
-
-## `g:ctrlspace_symbols`
-
-Enables you to provide your own symbols. It's useful if for example your font
-doesn't contain enough symbols or the glyphs are poorly rendered. 
-
-Default value:
-
-```VimL
-if g:ctrlspace_unicode_font
-  let g:ctrlspace_symbols = {
-        \ "cs":      "⌗",
- 
```

Comment:
Vim-Ctrlspace displays icons in the UI if your font supports UTF-8, or ASCII characters as a fallback. Some symbols (glyphs) might not look well with the font you are using, so feel free to change and adjust them.

This is the config I use for *Inconsolata* font in 

[your setup]

If you feel that you have found a better symbol for a given view, you are more than welcome to open a pull request.